src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { IconChevronDown } from '../Icons';
+
+import classNames from 'classnames';
+import styles from './Accordion.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface AccordionItem {
+  title: string;
+  text: string;
```

Comment:
Can we change this to `content: ReactNode`? I imagine there will be times where we want to put links/buttons/etc in the text, so will need to be `ReactNode` to handle those cases.

Reply:
Updated to ReactNode